README.md

```diff
@@ -14,36 +14,52 @@ Testing and benchmarking is done with `googletest` and `google/benchmark`.
 
 ## Getting started
 
-Clone the repository and initialize the submodules (googletest, benchmark).
+The respository is meant to be possible to build "out of the box", with standard
+CMake build procedures.
 
 ```shell
-git clone git@github.com:acts-project/detray.git
-cd detray
-git submodule update --init
+git clone https://github.com/acts-project/detray.git
+cmake -DCMAKE_BUILD_TYPE=Debug -S detray -B detray-build
```

Comment:
Should the ```Debug``` build really be the recommended one?

Reply:
I guess there's no good answer to this one.

My thinking was that when somebody clones the repository, they might be better served by a `Debug` build by default. For dealing with "using" this project, we might want to write an `INSTALL.md` file for it. Though at this point that would probably be way too premature.

I don't feel too strongly about it though. As long as @beomki-yeo will use `Debug` while developing the CUDA code. :smile:

Reply:
I see your point, but in general it should not happen that people accidentially built detray in debug and possible observe terrible performance because of it

Reply:
I am okay with either way :p...

Reply:
I think it's okay because it is just recommendation